### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kirstybutler**
Updated the requirements for project setup and further info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+Version 4.2 > is required to compile this project. From the latest changes, Arctic Fox (Canary 2, Beta) is required due to using Compose.
+
+### Xcode
+For running the iOS app via Xcode and for using the simulator, version 12.1 > is required.
+
+### Google Maps Api
+To use the location services and to view the available meals on the map, you will need to add the api key to your project.
+
+### Firebase Console
+We're using Firebse for authentication purposes - for viewing this data and making changes you'll need access to it.
```
Comment (fmontesino) [inline README.md:20]: Who should I tap up?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+## Prerequisites
+
+### Android Studio
+Version 4.2 > is required to compile this project. From the latest changes, Arctic Fox (Canary 2, Beta) is required due to using Compose.
+
+### Xcode
+For running the iOS app via Xcode and for using the simulator, version 12.1 > is required.
+
+### Google Maps Api
+To use the location services and to view the available meals on the map, you will need to add the api key to your project.
```
Comment (fmontesino) [inline README.md:17]: Where can I get the API key?

File: README.md [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
+## The concept
+"CommunityKitchen" is a platform to donate and find excess food for those in need that would otherwise go to waste. 
+Currently, a user can: create an account and login and search for meals in their local area. 
 
-### Installing json-server
-We are currently using `json-server` as a mock server to run the project against.
+## Prerequisites
+
+### Android Studio
+Version 4.2 > is required to compile this project. From the latest changes, Arctic Fox (Canary 2, Beta) is required due to using Compose.
```
Comment (fmontesino) [inline README.md:11]: Java 11 is also required I think

File: README.md (hunk 1 above)
Comment (kirstybutler) [reply README.md:20]: With the members changing constantly wasn't sure who to put exactly

File: README.md (hunk 2 above)
Comment (kirstybutler) [reply README.md:17]: Same as the Firebase project, someone can provide it :)

File: README.md (hunk 1 above)
Comment (fmontesino) [reply README.md:20]: Maybe the slack channel?

File: README.md (hunk 2 above)
Comment (fmontesino) [reply README.md:17]: So maybe it's worth mentioning where should I ask in order to get access to those files?

Comment (roger-tan) [reply README.md:17]: @Gmiller290488 gave me access to Firebase. 
For Google Maps, I can give access to Google Cloud Console

**Review by fmontesino — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (raynelle03) [reply README.md:17]: @fmontesino have added you to firebase.. Can you check if you have got access

**Review by raynelle03 — APPROVED** [review]

**Pull request merged**